### PR TITLE
Code coverage for Xtext editor server

### PR DESCRIPTION
**Pull request by barnettwilliam**
Enabled coverage for the Xtext editor server code. This can be run locally using the command `npm run coverage`. The unit tests and coverage are configured to run as part of a GitHub action on pushing to a repository. The coverage report is shown on the summary page of the completed GitHub action.

Part of mdenet/platformtools#54

### CODE REVIEW

File: xtext/editorserver/spec/support/jasmine.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "spec_dir": "spec",
+  "spec_files": [
+    "**/*[sS]pec.?(m)js"
+  ],
+  "helpers": [
+    "helpers/**/*.?(m)js"
+  ],
+  "env": {
+    "stopSpecOnExpectationFailure": false,
```
Comment (szschaler) [inline xtext/editorserver/spec/support/jasmine.json:10]: Why not?

File: xtext/editorserver/spec/testXtextControllerSpec.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+
+import {XtextController} from "../src/controllers/XtextController.js"
+import multer from 'multer';
+
+describe("XtextController", () => {
+
+    it("XtextController can be created", () => {
```
Comment (szschaler) [inline xtext/editorserver/spec/testXtextControllerSpec.js:7]: A fairly minimal, but serviceable test for now :-)

**Review by szschaler — APPROVED** [review]
LGTM, just one question which you're free to go either way on...

File: xtext/editorserver/spec/support/jasmine.json (hunk 1 above)
Comment (barnettwilliam) [reply xtext/editorserver/spec/support/jasmine.json:10]: This ensures the test continues when there are multiple assertions. Can be useful for debugging as you can see more information in the test report. Overall the test will fail.

**Pull request merged**